src/locale/locales/de/messages.po

```diff
@@ -2162,217 +2087,212 @@ msgstr "Diese Warnung ist nur für Beiträge mit angehängten Medien verfügbar.
-msgstr "Verwandlungen"
+msgstr ""
 
 #: src/view/com/post-thread/PostThreadItem.tsx:704
 #: src/view/com/post-thread/PostThreadItem.tsx:706
 #: src/view/com/util/forms/PostDropdownBtn.tsx:111
 msgid "Translate"
-msgstr "Übersetze"
+msgstr "Übersetzen"
 
 #: src/view/com/util/error/ErrorScreen.tsx:73
 msgid "Try again"
-msgstr "Nochmals versuchen"
+msgstr "Erneut versuchen"
 
 #: src/view/screens/ProfileList.tsx:472
 msgid "Un-block list"
-msgstr "Liste freigeben"
+msgstr ""
 
 #: src/view/screens/ProfileList.tsx:457
 msgid "Un-mute list"
-msgstr "Stummschaltliste aufheben"
+msgstr ""
 
 #: src/view/com/auth/create/CreateAccount.tsx:64
 #: src/view/com/auth/login/Login.tsx:76
 #: src/view/com/auth/login/LoginForm.tsx:117
 msgid "Unable to contact your service. Please check your Internet connection."
-msgstr "Sie können Ihren Dienst nicht kontaktieren. Bitte überprüfen Sie Ihre Internetverbindung."
+msgstr ""
 
 #: src/view/com/profile/ProfileHeader.tsx:466
 #: src/view/com/profile/ProfileHeader.tsx:469
 msgid "Unblock"
-msgstr "Freischalten"
+msgstr ""
 
 #: src/view/com/profile/ProfileHeader.tsx:304
 #: src/view/com/profile/ProfileHeader.tsx:388
 msgid "Unblock Account"
-msgstr "Konto freischalten"
+msgstr ""
 
 #: src/view/com/util/post-ctrls/RepostButton.web.tsx:48
 msgid "Undo repost"
-msgstr "Umbuchung rückgängig machen"
+msgstr "Repost rückgängig machen"
 
 #: src/view/com/auth/create/state.ts:210
 msgid "Unfortunately, you do not meet the requirements to create an account."
-msgstr "Leider erfüllen Sie nicht die Voraussetzungen, um ein Konto zu erstellen."
+msgstr "Leider erfüllst du nicht die Voraussetzungen, um einen Account zu erstellen."
 
 #: src/view/com/profile/ProfileHeader.tsx:369
 msgid "Unmute Account"
-msgstr "Konto entstören"
+msgstr ""
 
 #: src/view/com/util/forms/PostDropdownBtn.tsx:157
 msgid "Unmute thread"
-msgstr "Thema aufheben"
+msgstr ""
 
 #: src/view/screens/ProfileList.tsx:440
 msgid "Unpin moderation list"
-msgstr "Moderation Liste"
+msgstr ""
 
 #: src/view/com/modals/UserAddRemoveLists.tsx:54
 msgid "Update {displayName} in Lists"
-msgstr "Aktualisieren Sie {displayName} in Listen"
+msgstr ""
 
 #: src/lib/hooks/useOTAUpdate.ts:15
 msgid "Update Available"
-msgstr "Aktualisierung verfügbar"
+msgstr "Update verfügbar"
 
 #: src/view/com/auth/login/SetNewPasswordForm.tsx:172
 msgid "Updating..."
 msgstr "Aktualisieren..."
 
 #: src/view/com/modals/ChangeHandle.tsx:453
 msgid "Upload a text file to:"
-msgstr "Laden Sie eine Textdatei hoch:"
+msgstr ""
 
 #: src/view/screens/AppPasswords.tsx:194
 msgid "Use app passwords to login to other Bluesky clients without giving full access to your account or password."
-msgstr "Verwende App-Passwörter, um dich bei anderen Bluesky-Clients anzumelden, ohne dass du vollen Zugang zu deinem Konto oder Passwort hast."
+msgstr "Verwende App-Passwörter, um dich bei anderen Bluesky-Clients anzumelden, ohne dass du vollen Zugriff auf deinen Account oder Passwort hast."
```

Comment:
`"Verwende App-Passwörter, um dich bei anderen Bluesky-Clients anzumelden, ohne dass diese vollen Zugriff auf deinen Account oder Passwort bekommen."` (old would mean that you yourself don't have access)